guides/common/modules/ref_acd_prerequisites.adoc

```diff
@@ -0,0 +1,5 @@
+[id="{context}_prerequisites"]
```

Comment:
I would just include this in the introduction. It's not enough info to stand alone.

Reply:
This feedback still stands?

Reply:
Fixed.